pkg/arvo/sur/sole.hoon

```diff
@@ -6,8 +6,9 @@
 ++  sole-action                                         ::  sole to app
   $%  ::  {$abo ~}                                      ::  reset interaction
       {$det sole-change}                                ::  command line edit
-      {$ret ~}                                         ::  submit and clear
-      {$clr ~}                                         ::  exit context
+      {$ret ~}                                          ::  submit and clear
+      {$clr ~}                                          ::  exit context
+      {$tab pos=@ud}                                    ::  tab complete
```

Comment:
Nit, but this mixes old and new mold syntax.

Reply:
fixed